com.unity.render-pipelines.universal/Runtime/XR/XRLayout.cs

```diff
@@ -1,26 +0,0 @@
-// Helper API to create custom XR layout
```

Comment:
Are you missing deleting the .meta for this script?

Reply:
The whole folder is gone, there should be no left-over meta files now.